commit e3104862b84eb4e2971c0dab3fb0b7389e0e43fa
Author: John Spray <jspray@redhat.com>
Date:   Tue Jul 26 13:58:04 2016 +0100

    mgrc: fix report sending after failover
    
    The send_report loop doesn't get kicked off
    on configure messages unless the period is
    set to its initial value (0).
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index 715d68307f8..2a85f0df308 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -96,6 +96,7 @@ bool MgrClient::handle_mgr_map(MMgrMap *m)
                     << session->con->get_peer_addr() << dendl;
       delete session;
       session = nullptr;
+      stats_period = 0;
       if (report_callback != nullptr) {
         timer.cancel_event(report_callback);
         report_callback = nullptr;
